.github/workflows/go.yml

```diff
@@ -92,6 +97,15 @@ jobs:
           done
         working-directory: go/src/github.com/openconfig/ygot
 
+      - name: Staticcheck
```

Comment:
I wonder whether we should use this action going forward? https://github.com/grandcolline/golang-github-actions

It looks like there are a whole host of things that we'd get there which might give us additional coverage w.r.t static analysis? I'm happy to change to this after we merge this change.

Reply:
I'm not sure because our actions run on only certain directories, but the implementation there runs on `./...`: https://github.com/grandcolline/golang-github-actions/blob/master/entrypoint.sh#L201

So maybe it's not flexible enough?

Reply:
Ah, that makes sense. SGTM.